src/catch2/catch_test_case_info.hpp

```diff
@@ -112,6 +112,14 @@ namespace Catch {
         TestCaseHandle(TestCaseInfo* info, ITestInvoker* invoker) :
             m_info(info), m_invoker(invoker) {}
 
+        void testCaseStarting() const { 
```

Comment:
Not a big fan of this name. It is consistent with the event names, but it does not describe the reason why it exists well. How about "prepareTestCase" instead?

(And smae for `testCaseEnding` below.)

Reply:
Yup as below, I have renamed these prepareTestCase and tearDownTestCase